pkgs/servers/monitoring/sensu-go/default.nix

```diff
@@ -0,0 +1,54 @@
+{ buildGoPackage, fetchFromGitHub, lib }:
+
+let
+  generic = { subPackages, pname }:
+    buildGoPackage rec {
+      inherit pname;
+      version = "5.9.0";
+      shortRev = "078f625"; # for internal version info
+
+      goPackagePath = "github.com/sensu/sensu-go";
+
+      src = fetchFromGitHub {
+        owner = "sensu";
+        repo = "sensu-go";
+        rev = version;
+        sha256 = "1rivnq7m4p44zz1fl46j06aakb0yjsjb3mjqyfq4r0235xr01ajw";
+      };
+
+      goDeps = ./deps.nix;
```

Comment:
upstream vendorizes its dependencies, hence `goDeps` is not needed.
Besides that, LGTM